mlrun/projects/project.py

```diff
@@ -2021,6 +2009,7 @@ def run(
         artifact_path = artifact_path or self._enrich_artifact_path_with_workflow_uid()
 
         if schedule:
+            workflow_spec.overwrite = overwrite or workflow_spec.overwrite
             # Schedule = True -> use workflow_spec.schedule
             if not isinstance(schedule, bool):
                 workflow_spec.schedule = schedule
```

Comment:
?

Reply:
if the user pass `schedule=True` it means: "use the schedule that is defined in the workflow_spec", in this way the user can decide wether to use the schedule in the workflow_spec, pass a schedule of its own, or to just run the workflow (without schedule) with `schedule=False`

Reply:
you changed the wrong line, these 3 are not understandable in my eyes.